pom.xml

```diff
@@ -28,6 +28,19 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
 
+    <repositories>
```

Comment:
Why do we need to specify the snapshots repository? I'm surprised the snapshots have been working at all if this is needed?

Reply:
It's needed if you don't have that intoto4j snapshot dependency built locally.